app/models/host/discovered.rb

```diff
@@ -137,33 +137,49 @@ def proxied?
     subnet.present? && subnet.discovery.present?
   end
 
-  def proxy_url
-    proxied? ? subnet.discovery.url + "/discovery/#{self.ip}" : "https://#{self.ip}:8443"
+  def proxy_url(ip)
```

Comment:
`ip` shadows the ip attribute of the model. Can we be more specific here?

Reply:
Yeah good catch, I actually had `node_ip` originally there.